e2e/scripts/st_map.py

```diff
@@ -27,6 +27,7 @@
 
 # Cast is needed due to mypy not understanding the outcome of dividing
 # an array by a list of numbers.
+np.random.seed(1000)
```

Comment:
nit: It's best to call this directly under the imports with seed == 0 (any seed is theoretically fine, but I think we use 0 in all our tests), e.g.:

https://github.com/streamlit/streamlit/blob/9201a1980301a0cd62a5937982c410df08847a2f/e2e/scripts/st_legacy_dataframe_dimension_spec.py#L21

Reply:
Perfect! Updated in [e0b87fe](https://github.com/streamlit/streamlit/pull/5479/commits/e0b87fe1476734f996ee00fdeb029b60cbec021e)